File: set_1/easy/javascript/3c3a5515-7d4b-4950-8036-4660be6ad28b.js
// This is the input:
const input = [10, 4, 3, 50, 23, 90]

// Solution 1:
const arr = input;
const sortedArr = arr.sort((elem1, elem2) => {
    return elem1 - elem2
});

console.log(sortedArr.splice(sortedArr.length - 3, 3))

// Solution 2:

File: set_1/easy/python/3c3a5515-7d4b-4950-8036-4660be6ad28b.py

# Solution 1:
def print_largest_three(list):
    list.sort()
    list_len = len(list)
    return list[list_len-3:list_len]

list1 = [ 11, 65, 193, 36, 209, 664, 32 ];
res1 = print_largest_three(list1)
print(res1)

# Solution 2:
